packages/@coorpacademy-app-review/src/types/common.ts

```diff
@@ -147,15 +147,20 @@ export type CorrectionFromAPI = {
   corrections: CorrectedChoice[];
 };
 
-export type Skill = {
+export type SkillsToReview = {
```

Comment:
est-ce que tu peux mettre ça en singulier stp? skill**s** donne une idée de collection plutôt
```suggestion
export type SkillToReview = {
```

Reply:
@StefanoA1 ça marche